parm/config/gfs/config.ufs

```diff
@@ -256,6 +256,9 @@ if [[ "${skip_mom6}" == "false" ]]; then
       MOM6_RESTART_SETTING='r'
       MOM6_RIVER_RUNOFF='False'
       eps_imesh="4.0e-1"
+      MOM6_DIAG_COORD_DEF_Z_FILE="oceanda_zgrid_25L.nc"
+      MOM6_DIAG_MISVAL="0.0"
+      MOM6_ALLOW_LANDMASK_CHANGES='False'
```

Comment:
Does this also need to be added to the 50 and 025 sections?  

And since I'm looking @aerorahul should the 50 section actually be 050?

Reply:
yes. `50` should be `050`. 
I was going to raise the question of `1/2-deg` ocean resolution.  Is this something that is useful?  Should we keep it/remove it.  We don't have all the files for this resolution (not that I know of).

Reply:
add these vars to the other resolutions, also changed 50 to 050